tests/src/Kernel/CollaboraMediaAccessTest.php

```diff
@@ -18,58 +18,21 @@
 use Drupal\Core\Session\AccountInterface;
 use Drupal\Core\Session\AnonymousUserSession;
 use Drupal\Core\Url;
-use Drupal\KernelTests\KernelTestBase;
-use Drupal\Tests\collabora_online\Traits\MediaCreationTrait;
-use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
-use Drupal\Tests\user\Traits\UserCreationTrait;
 use Drupal\user\RoleInterface;
 
 /**
  * Tests access for media operations and routes.
  */
-class CollaboraMediaAccessTest extends KernelTestBase {
-
-  use MediaTypeCreationTrait;
-  use UserCreationTrait;
-  use MediaCreationTrait;
-
-  /**
-   * {@inheritdoc}
-   */
-  protected static $modules = [
-    'collabora_online',
-    'user',
-    'media',
-    'field',
-    'image',
-    'system',
-    'file',
-  ];
+class CollaboraMediaAccessTest extends CollaboraKernelTestBase {
 
   /**
    * {@inheritdoc}
    */
-  protected function setUp(): void {
+  public function setUp(): void {
```

Comment:
Why this change of visibility?

Reply:
I removed this and then re-added manually with wrong visibility. Good catch!